net/poll/src/fallible.rs

```diff
@@ -1,8 +1,8 @@
 use std::sync::Mutex;
 
-lazy_static! {
-    pub(super) static ref FALLIBLE: Mutex<Option<f64>> = Mutex::new(None);
-}
+use once_cell::sync::Lazy;
+
+pub(super) static FALLIBLE: Lazy<Mutex<Option<f64>>> = Lazy::new(|| Mutex::new(None));
```

Comment:
Can we actually remove the `Lazy` here, given that `Mutex::new` is `const`?

Reply:
Yeah! I thought about it while I did the refactoring, and also it worked pretty well, thanks!

But I am not sure if I catch your suggestion on the const! Do you mean `Mutex::new` is const or to make `FALLIBLE` const?